TestCases/parallel_regression.py

```diff
@@ -185,6 +185,17 @@ def main():
     bluntbody.tol       = 0.00001
     test_list.append(bluntbody)
 
+    # Equivalent area NACA64-206
+    ea_naca64206           = TestCase('ea_naca64206')
+    ea_naca64206.cfg_dir   = "optimization_euler/equivalentarea_naca64206"
+    ea_naca64206.cfg_file  = "NACA64206_direct.cfg"
+    ea_naca64206.test_iter = 20
+    ea_naca64206.test_vals = [-1.568224, 1.6373e-08]
```

Comment:
More .test_vals i.e. more sreen_output for better overall coverage. Please use SU2_CFD -d <your.cfg> see available options and pack in a few more things that you feel comfortable with and think their respective values are reasonable (I am not exactly knowledgable when it comes to supersonic stuff)

Reply:
I have increased the number of variables in `.test_vals` to 4.